queue_services/entity-emailer/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "entity-emailer"
+version = "3.1.0"
+description = "This module is the service worker for sending emails about entity related events."
+authors = ["leodube <leo@dube.ca>"]
+license = "BSD-3-Clause"
+readme = "README.md"
+packages = [{include = "entity_emailer", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+Flask = "^2.3.2"
+python-dotenv = "^1.0.0"
+sentry-sdk = "^1.27.1"
+launchdarkly-server-sdk = "^8.1.4"
+legal_api = { git = "https://github.com/bcgov/lear.git", subdirectory = "legal-api", branch = "dev_legal_name_changes" }
+simple_cloudevent = { git = "https://github.com/daxiom/simple-cloudevent.py" }
+requests = "^2.31.0"
+google-auth = "^2.21.0"
+google-cloud-pubsub = "^2.17.1"
+pg8000 = "^1.29.8"
```

Comment:
I know we pull in the `jinja2` dependency via the `legal_api` module, but can we import it explicitly here?  

Just want to make that explicit for anything emailer actually uses.

Can you go through and see if there's any other dependency like that?

Reply:
That makes sense, I'll have a look to see if there are any other dependencies that the emailer uses that are not explicitly imported.